### ISSUE TITLE
Setting headers

### ISSUE THREAD

**Issue by shornuk** [sjelfull/craft3-beam#6]
Hi, 

Nice little plugin. Using it as per the docs, I've had to add the following to the template

```
{% header "Content-Type: application/octet-stream" %}
{% header "Content-Transfer-Encoding: binary" %}
```
when exporting a csv, otherwise Safari adds `.html` on the downloaded file. 

Not sure if this should be handled by the plugin or worth just adding to docs. Cheers!

**Comment by sjelfull** [comment]
Hey, thanks for this. Doing things which requires a correctly formed output is not optimal, so I have planned some improvements to this once I have some time.

Optimally you would just call `{% do beam.setHeader(data) %}` `{% do beam.addRow(data) %}`, `{% do beam.download(config) %}` etc. without having to fumble with manual headers and the like.

**Comment by shornuk** [comment]
Sounds good 👍🏼

**Comment by sjelfull** [comment]
Should be fixed shortly when 2.1 is released. It will redirect to another url and set the headers there instead of using `text/html` which the template outputs.
